### DISCUSSION TITLE
PHP Parse error: email.php on line 53

### DISCUSSION THREAD

**Discussion by shinas111** [maxsite/cms#491] [Q&A] [answered]
Здравствуйте! 500ошибка, 
В логах 
PHP Parse error:  syntax error, unexpected '?' in /application/maxsite/common/core/email.php on line 53

// (переделка из-за ужесточения правил почтовиков)
	// $from теперь это reply-to: 
	// а from: всегда равен admin_email_server из настроек сайта
	$admin_email = mso_get_option('admin_email_server', 'general', 'admin@site.com');
	$from_name = $preferences['from_name'] ?? ''; //getinfo('name_site');

**Comment by maxsite** [comment, marked as answer]
Версия php должна быть более 7.1

**Comment by shinas111** [comment]
Спасибо
